oaipmh/requests/routes.py

```diff
@@ -41,6 +40,27 @@ def oai() -> Response:
 
     return response, code, headers
 
+@blueprint.route("/OAI/arXivRaw.xsd", methods=['GET', 'POST'])
+def schema_arXivRaw() -> Response:
+    response=render_template("schema/arXivRaw.xsd")
```

Comment:
I is probably more efficient to do https://flask.palletsprojects.com/en/stable/api/#flask.send_file